zerver/models.py

```diff
@@ -950,7 +950,15 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     dense_mode: bool = models.BooleanField(default=True)
     fluid_layout_width: bool = models.BooleanField(default=False)
     high_contrast_mode: bool = models.BooleanField(default=False)
-    night_mode: bool = models.BooleanField(default=False)
+    AUTOMATIC_THEME_DETECTION = 1
+    NIGHT_MODE = 2
+    LIGHT_MODE = 3
+    COLOR_SCHEME_CHOICES = [
+        AUTOMATIC_THEME_DETECTION,
+        NIGHT_MODE,
+        LIGHT_MODE
+    ]
+    color_scheme = models.PositiveSmallIntegerField(default=AUTOMATIC_THEME_DETECTION)
```

Comment:
Can you move this away from the block of boolean fieds, for readability?  And we should also prefix the variable names as e.g. `COLOR_SCHEME_AUTOMATIC`, `COLOR_SCHEME_NIGHT`, etc., like we do with our other enum format values.